dstep/translator/MacroDefinition.d

```diff
@@ -100,6 +100,30 @@ struct ExpressionContext
         result.params = params;
         return result;
     }
+
+    void elevateImports()
+    {
+        assert(imports != null);
+
+        foreach (item; imports.byKey)
+            context.includeHandler.addImport(item);
+    }
+}
+
+string translate(Literal literal, ExpressionContext context)
+{
+    import std.algorithm;
+    import std.ascii;
+
+    if (literal.spelling.length > 1 &&
+        all!isDigit(literal.spelling) &&
+        literal.spelling.front == '0')
+    {
+        (*context.imports).add("std.conv : octal");
+        return "octal!" ~ literal.spelling[0..$-1].stripLeft('0') ~ literal.spelling[$-1];
```

Comment:
Doesn't look like unsigned and long versions of the literals are handled properly, i.e. `0700u` and `0700L`.

Reply:
Fixed.